core/DependencyInjection/ContainerBuilder.php

```diff
@@ -28,7 +28,12 @@
 use Psr\Container\ContainerInterface;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder as SymfonyContainerBuilder;
+use Symfony\Component\DependencyInjection\ContainerInterface as SynfonyContainerInterface;
```

Comment:
![image](https://user-images.githubusercontent.com/59471572/135105479-54568ee4-8f2a-4c8a-9a64-80a1d680e77c.png)
```suggestion
use Symfony\Component\DependencyInjection\ContainerInterface as SymfonyContainerInterface;
```

Reply:
done :)